### PR TITLE
clean up pkg/cluster/config

### PR DESCRIPTION
**Pull request by alejandrox1**
/kind cleanup

Signed-off-by: Jorge Alarcon Ochoa <alarcj137@gmail.com>

### CODE REVIEW

File: pkg/cluster/config/register.go [hunk 1]
```diff
@@ -21,18 +21,21 @@ import (
...
 
 var (
-	// SchemeGroupVersion is group version used to register these objects
+	// SchemeGroupVersion is group version used to register these objects.
 	SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: "v1alpha1"}
+	// SchemeBuilder is a type to collect functions that add things to a schema.
 	SchemeBuilder      runtime.SchemeBuilder
 	localSchemeBuilder = &SchemeBuilder
-	AddToScheme        = localSchemeBuilder.AddToScheme
+	// AddToScheme applies all the sotred functions in the localSchemeBuilder
```
Comment (munnerz) [inline pkg/cluster/config/register.go]: `s/sotred/sorted`

File: pkg/cluster/config/register.go [hunk 2]
```diff
@@ -21,18 +21,21 @@ import (
...
 
-// GroupName is the group name used in this package
+// GroupName is the group name used in this package.
 const GroupName = "kind.sigs.k8s.io"
 
 var (
-	// SchemeGroupVersion is group version used to register these objects
+	// SchemeGroupVersion is group version used to register these objects.
 	SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: "v1alpha1"}
+	// SchemeBuilder is a type to collect functions that add things to a schema.
```
Comment (munnerz) [inline pkg/cluster/config/register.go]: `s/things/API types`? 'things' feels a bit wishy-washy 😄

File: pkg/cluster/config/default.go [hunk 3]
```diff
@@ -20,24 +20,27 @@ import (
...
+// image.
 const DefaultImage = "kindest/node:v1.12.2@sha256:6ac1dc1750fc0efd13d4e294115f9012a21282957e4380a5535bd32154193d4d"
 
-// New returns a new default Config
 func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
 
-func SetDefaults_Config(obj *Config) {
+// SetDefaultsConfig sets uninitializex fields to their default value.
```
Comment (munnerz) [inline pkg/cluster/config/default.go]: `s/uninitializex/uninitialized`

File: pkg/cluster/config/types.go [hunk 4]
```diff
@@ -42,6 +42,8 @@ type Config struct {
 	ControlPlane *ControlPlane `json:"ControlPlane,omitempty"`
 }
 
+// ControlPlane holds node life cycle hooks to make the actual state of all
```
Comment (BenTheElder) [inline pkg/cluster/config/types.go]: Hmm I'd say something like above instead, ControlPlane holds configuration specific to the "control plane" node(s) (currently the only node)

File: pkg/cluster/config/register.go (hunk 1 above)
Comment (alejandrox1) [reply pkg/cluster/config/register.go]: haha it was actually supposed to be "stored."

File: pkg/cluster/config/types.go (hunk 4 above)
Comment (alejandrox1) [reply pkg/cluster/config/types.go]: change made

**Pull request merged**